src/sass/_contacts.scss

```diff
@@ -0,0 +1,168 @@
+.contacts{
+    font-family: 'DM Sans', sans-serif;
+
+    &__container{
+        padding-top: 20px;
+        background: var(--bottom-section-background);
```

Comment:
Brakuje czegoś, żeby "zepchnąć" kolor tła w dół. W Figmie, pierwsza karta wystaje w góre na inny kolor.

Reply:
Tutaj nie mam pomysłu jak mógłbym to poprawnie zepchnąć, miałbyś jakieś rozwiązanie (próbowałem background-position ale coś nie działa)? Jedynie to mi już zostało.